File: flutter_module/lib/userInfo.dart
class UserInfo {

  String account;
  String password;

  UserInfo(
    this.account,
    this.password
  );

  Map<String, dynamic> toMap() => {'account': account, 'password': password};

  @override
  String toString() {
    return "account: $account, password: $password";
  }

}

File: app/src/main/java/com/darrenyuen/webdavclient/bean/UserInfo.kt
package com.darrenyuen.webdavclient.bean

/**
 * Create by yuan on 2021/3/16
 */
data class UserInfo(val account: String, val password: String) {
    override fun toString(): String {
        return "account: $account, password: $password"
    }
}
